src/services/apiService.js

```diff
@@ -18,7 +18,8 @@ export async function GetAllLikedDisliked(userId) {
 }
 
 export async function PostReactionForUser(data, token) {
-  if (data == null) {
+  const isEmpty = Object.values(data).some((x) => x == null || x == '');
```

Comment:
Do this instead
const isEmpty = Object.values(data).some((x) => !x);

Reply:
Will that check if x is an empty string?

Reply:
Yes it does
https://stackoverflow.com/questions/6003884/how-do-i-check-for-null-values-in-javascript

Reply:
Ahh cool, I didn't know that, Thanks!